tensorflow/python/ops/image_ops_test.py

```diff
@@ -1331,6 +1331,30 @@ def testRot90NumpyEquivalenceWithBatch(self):
         y_np = np.rot90(image, k=k, axes=(1, 2))
         self.assertAllEqual(y_np, y_tf.eval({k_placeholder: k}))
 
+  def testRot90ND4Dims(self):
+    image = np.arange(48, dtype=np.uint8).reshape([2, 2, 4, 3])
+    with self.test_session(use_gpu=True):
+      rotated = image
+      for _ in xrange(4):
+        rotated = image_ops.rot90ND(rotated, 0, 1)
+      self.assertAllEqual(image, rotated.eval())
```

Comment:
Can we make this testing a bit more comprehesive? (For example, right now if `rot90ND` just returned the input as-is, this test would pass). See the tests above that use `np.rot90`.